src/universal/components/SignInPage/SignInEmailPasswordForm.js

```diff
@@ -32,25 +35,33 @@ const ForgotPasswordLink = styled(Link)({
   textAlign: 'center'
 });
 
+const noop = () => {};
+
 const SignInEmailPasswordForm = (props: Props) => (
-  <Form onSubmit={props.handleSubmit}>
+  <Form onSubmit={props.isSubmitting ? noop : props.handleSubmit}>
```

Comment:
i think you had it right the first time. redux-form checks `isSubmitting` internally, right?
(personally, I'm moving away from redux-form as much as possible because it complicates my code, but that's just my opinion).

Reply:
I dig having validation logic, but I think putting `redux` in the name was unnecessary.